### PR TITLE
feat : added avatar feature

### PR DESCRIPTION
**Pull request by vivekgudipudi**
added avatar feature and refactored alerts page

### CODE REVIEW

File: CSS/introduction.css [hunk 1]
```diff
@@ -39,7 +39,6 @@
     display: flex;
     align-items: center;
     justify-content: center;
```
Comment (nishasen) [inline CSS/introduction.css:41]: if this file is for your doc site, then use utility class here, then you have to right less code

**Review by nishasen — APPROVED** [review]
Your code looks clean and good, a small change suggested.

File: CSS/introduction.css (hunk 1 above)
Comment (vivekgudipudi) [reply CSS/introduction.css:41]: sure will change, thanks for suggestion

**Review by MiryalaAkshitha — APPROVED** [review]

**Comment by MiryalaAkshitha** [comment]
good use of utility classes.
your naming convention made review easy.

**Pull request merged**